test/cljs/status_im/test/models/network.cljs

```diff
@@ -96,8 +96,9 @@
 
 (deftest save
   (testing "it does not save a network with an invalid url"
-    (is (nil? (model/save {:random-id "random"
+    (is (nil? (model/save {:random-id-generator #(identity "random")
```

Comment:
`(constantly random)` is probably more idiomatic

Reply:
thanks I forgot about that one, I will replace it in #6019 once @janherich will have fixed conflicts with your recent merge